code/modules/mob/mob_defines.dm

```diff
@@ -106,6 +106,7 @@
 	var/candrop = 1
 	var/lastpuke = 0
 	var/unacidable = 0
+	var/nospells = 0 //Can't use spells. At all.
```

Comment:
Just use spell silencing, it's inbuilt to spells already, I think I made jaunting use it.